### PR TITLE
added it-CH locale

### PR DESCRIPTION
**Pull request by maic66**
Please be patient if I did some mistakes, it's my first PR contribution.

### CODE REVIEW

**Review by fturmel — CHANGES_REQUESTED** [review]
LGTM! See my comment about the missing jsdoc annotation.

**Review by maic66 — COMMENTED** [review]
added missing jsdoc annotation

File: src/locale/it-CH/index.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+import formatRelative from '../it/_lib/formatRelative/index'
+import localize from '../it/_lib/localize/index'
+import match from '../it/_lib/match/index'
+import formatLong from './_lib/formatLong/index'
+
+/**
+ * @type {Locale}
+ * @category Locales
+ * @summary Italian locale (Switzerland).
+ * @language Italian
```
[the remark this answers is not in the document]
Comment (maic66) [reply src/locale/it-CH/index.ts:12]: added missing jsdoc annotation

**Review by fturmel — APPROVED** [review]

**Comment by Spinnenzunge** [comment]
Thank you very much 💋

**Comment by maic66** [comment]
I've seen this was a late merge and it's still not reflected in npm package.
How does it work?

**Comment by fturmel** [comment]
@maic66 there just hasn't been an official release since the PR was merged. I don't know for sure, but it seems like there is only one task holding off the next release #2887

**Comment by maic66** [comment]
Hi all. I updated my npm packages but this commit is still not present.
Has somebody an explanation for this?

**Pull request merged**